test/integration/oauth_basicauth_test.go

```diff
@@ -288,7 +309,149 @@ func TestOAuthBasicAuthPassword(t *testing.T) {
 		t.Fatalf("Unexpected error: %v", err)
 	}
 	if user.Name != expectedUsername {
-		t.Fatalf("Expected username as the user, got %v", user)
+		t.Fatalf("Expected %v as the user, got %v", expectedUsername, user)
+	}
+
+}
+
+var redirectCodes = []int{300, 301, 302, 303, 304, 305, 306, 307}
+var redirectCodeIndex = 0
```

Comment:
I'd rather not move to static package-level vars to drive this behavior, updating the remote basic auth handler closure should be sufficient